### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Alyoninthecity**
To install all the requirements "brotlipy" requires "Microsoft Visual C++ 14.0+"

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Small fix to the documentation of the README.md

### CODE REVIEW

**Comment by cullzie** [comment]
Hey @Alyoninthecity 
Thanks for the contribution. 

Apparently it should just work on windows according to the documentation: https://python-hyper.org/projects/brotlipy/en/latest/installation.html
Based on this I think it should go under FAQ instead of requirements.

Can you describe the kind of error you were seeing?

**Comment by Alyoninthecity** [comment]
Screenshot (275)
So I had this problem during the installation. Fixed by downloading Microsoft Visual C++ 14.0+ (recreated the error by setting up a with a new virtual machine with Windows 10)

**Comment by Alyoninthecity** [comment]
> Hey @Alyoninthecity
> Thanks for the contribution.
> 
> Apparently it should just work on windows according to the documentation: https://python-hyper.org/projects/brotlipy/en/latest/installation.html
> Based on this I think it should go under FAQ instead of requirements.
> 
> Can you describe the kind of error you were seeing?

Screenshot (275)
So I had this problem during the installation. Fixed by downloading Microsoft Visual C++ 14.0+ (recreated the error by setting up a with a new virtual machine with Windows 10)

**Comment by fakeid30** [comment]
> Do you think this is a requirement though? I feel it might be more a FAQ if not everyone has to do it.
> I have seen a few people need to install Microsoft Visual C++ now but I have no idea if that is all cases for users on windows or just a subset of users

So far I have tested this seems to be the requirement for all users. I think we can merge this. Waiting your input.

File: README.md [hunk 1]
```diff
@@ -48,6 +48,10 @@ Also, don't forget to **Fork & Star the repository if you like it!**
 
 **Required Python version:** [Python 3.8+](https://www.python.org/downloads/)
 
+**Required Microsoft Visual C++ 14.0+ version:** [Microsoft Visual C++ 14.0+](https://visualstudio.microsoft.com/visual-cpp-build-tools/)
```
Comment (cullzie) [inline README.md:51]: I'd add something here to signify this is a windows only step.

**Comment by cullzie** [comment]
I'm happy to approve once there is something to show it is a windows only requirement. Might lead to more confusion otherwise

**Comment by Alyoninthecity** [comment]
> I'm happy to approve once there is something to show it is a windows only requirement. Might lead to more confusion otherwise

Hope like that sounds better, let me know

**Pull request merged**